var.tfvars

```diff
@@ -5,7 +5,7 @@ images_path     = "/home/libvirt/openshift-images"
 
 ### Configure the Nodes details
 bastion_image   = "http://remote_server/rhel-8.1-ppc64le-kvm.qcow2"
-rhcos_image     = "http://remote_server/rhcos-4.3.18-ppc64le-qemu.ppc64le.qcow2"
+rhcos_image     = "http://remote_server/rhcos-4.4.0-0.nightly-ppc64le-2020-05-08-093850-qemu.ppc64le.qcow2"
 bastion         = { memory = 4096, vcpu = 2 }
 bootstrap       = { memory = 8192, vcpu = 4, count = 1 }
 master          = { memory = 8192, vcpu = 4, count = 3 }
```

Comment:
Let's bump up the memory to 16GB for the default case

Reply:
Let me know if the new config looks good.